lib/activerecord_spanner_adapter/transaction.rb

```diff
@@ -75,6 +76,15 @@ def begin
       end
     end
 
+    def _transaction_isolation_level_to_grpc isolation
+      case isolation
+      when :serializable
+        Google::Cloud::Spanner::V1::TransactionOptions::IsolationLevel::SERIALIZABLE
+      when :repeatable_read
+        Google::Cloud::Spanner::V1::TransactionOptions::IsolationLevel::REPEATABLE_READ
+      end
+    end
```

Comment:
Note: This does not return anything for any unknown/unsupported isolation level. This is intentional, as that was also the behavior before introducing support for isolation levels. That is: Calling this method with a random isolation level would just cause the isolation level to be ignored.